opentelemetry-user-events-logs/examples/basic.rs

```diff
@@ -24,15 +24,9 @@ fn main() {
     let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
     tracing_subscriber::registry().with(layer).init();
 
-    // event_name is now passed as an attribute, but once https://github.com/tokio-rs/tracing/issues/1426
-    // is done, it can be passed with name:"my-event-name", so it'll be available as metadata for
-    // fast filtering.
-    // event_id is also passed as an attribute now, there is nothing in metadata where a
-    // numeric id can be stored.
     error!(
-        event_name = "my-event-name",
```

Comment:
Consider keeping this in a comment (just show what we **intend** to have in the end).

Reply:
Good point. Done.